x-pack/plugins/uptime/public/components/queries/ping_list/ping_list.tsx

```diff
@@ -50,9 +52,24 @@ export class Pings extends React.Component<PingListProps, PingListState> {
     super(props);
 
     const statusOptions = [
-      { label: 'All', value: '' },
-      { label: 'Up', value: 'up' },
-      { label: 'Down', value: 'down' },
+      {
+        label: i18n.translate('xpack.uptime.pingList.statusOptions.allStatus', {
+          defaultMessage: 'All',
+        }),
+        value: '',
+      },
+      {
+        label: i18n.translate('xpack.uptime.pingList.statusOptions.upStatus', {
+          defaultMessage: 'Up',
+        }),
+        value: 'up',
+      },
+      {
+        label: i18n.translate('xpack.uptime.pingList.statusOptions.allStatus', {
```

Comment:
This should be `.downStatus`

Reply:
Copy/paste strikes again ✂️

Reply:
Addressed in https://github.com/elastic/kibana/pull/28183/commits/994eb8cf052e8b0e41010e58135eef94d92163e7